ontologies/vocabs.ttl

```diff
@@ -466,8 +460,10 @@ While a Threat Actor or Intrusion Set may be seeking personal gain, this does no
 
 gist:_AttackMotivation_personal-satisfaction
 	a gist:AttackMotivation ;
-	rdfs:label "Personal Satisfaction"^^xsd:string ;
-	skos:altLabel "Personal-Satisfaction"^^xsd:string ;
+	skos:altLabel
+		"Personal Satisfaction"^^xsd:string ,
+		"Personal-Satisfaction"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?